modules/core/src/shaderlib/project/project.glsl.js

```diff
@@ -134,6 +134,11 @@ vec4 project_position(vec4 position, vec2 position64xyLow) {
   // METER_OFFSETS or IDENTITY
   // Apply model matrix
   vec4 position_modelspace = project_uModelMatrix * position;
```

Comment:
Unrelated, but this seems wrong. Multiplying a position in model space with the project_uModelMatrix should give us world coordinates..., right?

Reply:
...that is correct